specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -1535,6 +1539,48 @@
         }
       }
     },
+    "NetworkRuleSet": {
```

Comment:
What is the reason that networkRuleset is only defined in this version and not the 2018-09-01 version? Is it now obsolete?

Reply:
`2018-09-01 ` doesn't have this resource type, `2017-10-01` is the latest version that has `registry`.